### PR TITLE
Adding Footer + CSS styles

### LINKED ISSUE #4: Develop Footer with Linked 'Designed & Developed by Bizznest'
#### Description
Create a footer section for the website containing the text 'Designed & Developed by Bizznest' linked to Bizznest's website.

#### Tasks
- [x] Create HTML/CSS structure for the footer section.
- [x] Ensure the 'Designed & Developed by Bizznest' text is appropriately linked to [bizznest.digitalnest.org](https://bizznest.digitalnest.org).
- [x] Ensure the footer is responsive and compatible with various screen sizes.

### PR DESCRIPTION
**Pull request by elimanzo**
footer

closes #4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css" /> 
-    <title>Document</title>
-</head>
-<body>
+<html>
+  <head>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <link
```
Comment (alexrdrgz) [inline index.html:4]: I would keep this on the same line and not break it up, unless you're going to do that with the other links.

Over all, it's more of a personal preference but whatever you choose should be consistent through out.

**Review by alexrdrgz — CHANGES_REQUESTED** [review]
Good job with branch naming and commits this is all looking WAY better.

Now i'm just nit picking but this is looking good,minor changes requested

File: index.html (hunk 1 above)
Comment (elimanzo) [reply index.html:4]: So i believe it's prettier formatting this for me, and I have it setup where every time I save it auto formats to it, but I could disable this for now.

**Review by alexrdrgz — APPROVED** [review]
Looks great Good Job!

**Pull request merged**